lib/middlewares/src/metering.rs

```diff
@@ -183,6 +240,16 @@ pub fn get_remaining_points(instance: &Instance) -> u64 {
         .expect("`remaining_points` from Instance has wrong type")
 }
 
+pub fn get_error(instance: &Instance) -> MeteringError {
+    instance
+        .exports
+        .get_global("error_code")
+        .expect("Can't get `error_code` from Instance")
```

Comment:
Same here, this should mention the context (metering). Not necessarily if the global variable name is changed to include the word "metering"